### PR TITLE
Add topic about how to update product attribute in bulk

### PR DESCRIPTION
**Pull request by EvheniiaKolkhiieva**
## Purpose of this pull request

This pull request (PR) adds steps to update product attribute in bulk

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

- https://docs.magento.com/user-guide/catalog/product-attributes.html

whatsnew
Added the [Bulk Updates for Product Attributes](https://docs.magento.com/user-guide/catalog/bulk-product-attribute-update.html) topic, which describes how to update product attributes across a group of selected products.

### CODE REVIEW

File: src/stores/bulk-product-attribute-update.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: Bulk Updating Attributes
+---
+Use the "Update Attributes" tool to change one or more attributes in your products. This tool helps to massively transform the desired parts of the products.
```
Comment (leslietilling) [inline src/stores/bulk-product-attribute-update.md:4]:
```suggestion
Use the _Update Attributes_ tool to change one or more attributes in your products. This tool allows you to apply significant changes across a large group of products.
```

File: src/stores/bulk-product-attribute-update.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+1. Select the products for which you want to modify sources.
+
+   Browse or search to find the products and select those checkboxes.
+
+1. Click the **Actions** menu at the top and choose **Update Attributes**.
+
+    ![]({% link images/images/bulk-product-updating-action.png %}){: .zoom}
+    _Select products to attribute update_
+
+1. Update the attribute, advanced inventory or website data for product you want.
```
Comment (leslietilling) [inline src/stores/bulk-product-attribute-update.md:19]:
```suggestion
1. Update the attribute, advanced inventory, or website data for the selected products, according to your needs.
```

File: src/stores/bulk-product-attribute-update.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+1. When finish, click <span class="btn">Save</span>.
```
Comment (leslietilling) [inline src/stores/bulk-product-attribute-update.md:21]:
```suggestion
1. When finished, click <span class="btn">Save</span>.
```

File: src/stores/bulk-product-attribute-update.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+    ![]({% link images/images/bulk-pproduct-attribute-update.png %}){: .zoom}
+    _Bulk Updating Attribute_
```
Comment (leslietilling) [inline src/stores/bulk-product-attribute-update.md:24]:
```suggestion
    _Bulk updating for attributes_
```

File: src/stores/bulk-product-attribute-update.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+1. On the _Admin_ sidebar, go to **Catalog** > **Products**.
+
+1. Select the products for which you want to modify sources.
+
+   Browse or search to find the products and select those checkboxes.
+
+1. Click the **Actions** menu at the top and choose **Update Attributes**.
+
+    ![]({% link images/images/bulk-product-updating-action.png %}){: .zoom}
+    _Select products to attribute update_
```
Comment (leslietilling) [inline src/stores/bulk-product-attribute-update.md:17]:
```suggestion
    _Select products to be updated_
```

File: src/stores/bulk-product-attribute-update.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
+---
+title: Bulk Updating Attributes
```
Comment (leslietilling) [inline src/stores/bulk-product-attribute-update.md:2]:
```suggestion
title: Bulk Updates for Product Attributes
```

File: src/_data/toc/catalog.yml [hunk 7]
```diff
@@ -276,6 +276,8 @@ pages:
       url: "/catalog/product-attributes-add.html"
   - label: Deleting Product Attributes
     url: "/stores/attribute-delete.html"
+  - label: Bulk Updating Attributes
```
Comment (leslietilling) [inline src/_data/toc/catalog.yml:279]:
```suggestion
  - label: Bulk Updates for Product Attributes
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
minor edits

**Comment by EvheniiaKolkhiieva** [comment]
Hi @leslietilling , thank you for reviewing. I have implemented your edits, could you please approve this PR?
Thank you! :)

File: src/_data/toc/catalog.yml (hunk 7 above)
Comment (EvheniiaKolkhiieva) [reply src/_data/toc/catalog.yml]: Done ✅

File: src/stores/bulk-product-attribute-update.md (hunk 6 above)
Comment (EvheniiaKolkhiieva) [reply src/stores/bulk-product-attribute-update.md]: Done ✅

File: src/stores/bulk-product-attribute-update.md (hunk 5 above)
Comment (EvheniiaKolkhiieva) [reply src/stores/bulk-product-attribute-update.md]: Done ✅

File: src/stores/bulk-product-attribute-update.md (hunk 4 above)
Comment (EvheniiaKolkhiieva) [reply src/stores/bulk-product-attribute-update.md]: Done ✅

File: src/stores/bulk-product-attribute-update.md (hunk 3 above)
Comment (EvheniiaKolkhiieva) [reply src/stores/bulk-product-attribute-update.md]: Done ✅

File: src/stores/bulk-product-attribute-update.md (hunk 2 above)
Comment (EvheniiaKolkhiieva) [reply src/stores/bulk-product-attribute-update.md]: Done ✅

File: src/stores/bulk-product-attribute-update.md (hunk 1 above)
Comment (EvheniiaKolkhiieva) [reply src/stores/bulk-product-attribute-update.md]: Done ✅

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by dobooth** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@dobooth the pull request successfully imported.

**Pull request merged**